filer/templates/admin/filer/breadcrumbs.html

```diff
@@ -11,7 +11,7 @@
             {% if ancestor_folder.label %}
                 {{ ancestor_folder.label }}
             {% else %}
-                {% if ancestor_folder.name != "unfiled files" %}
+                {% if ancestor_folder.name != _("Unsorted Uploads") %}
```

Comment:
@cyberstar why we can't use `_()` in templates? https://docs.djangoproject.com/en/1.9/topics/i18n/translation/#string-literals-passed-to-tags-and-filters